server/sqlflowserver.go

```diff
@@ -46,12 +46,24 @@ type Server struct {
 
 // Run implements `rpc Run (Request) returns (stream Response)`
 func (s *Server) Run(req *pb.Request, stream pb.SQLFlow_RunServer) error {
-	pr := s.run(req.Sql, s.db, s.modelDir)
+	db := s.db
+	var err error
+
+	// TOOD(Yancey1989): delete the IF statment
+	// when we enable session in the pysqlflow repo by default.
+	if req.Session != nil {
```

Comment:
Maybe add a global settings variable when starting the sqlflowserver process.

Reply:
Maybe we can enable session when the Flag `--datasource` is an empty string, that we don't need another new flag.